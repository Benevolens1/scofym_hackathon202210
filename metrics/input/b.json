{"1":{
	"type": "typeA",
	"arguments": {"arg1": "Abra",
		"arg2": "Kadabra"}
	},
"2": {
	"type": "typeB",
	"arguments": {"arg1": "Huhu"}
},
"3":{
	"type": "typeA",
	"arguments": {"arg1": "Happy",
		"arg2": "new",
        "arg3": "year"}
	},
"4":{
    "type": "typeB",
    "arguments": {}
}
}
