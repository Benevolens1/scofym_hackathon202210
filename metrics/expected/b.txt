1 Abra_Kadabra
2 Huhu
3 Happy_newyear
4 Not filled !
